### DISCUSSION TITLE
windows ARM on Mac M1 using UTM

### DISCUSSION THREAD

**Discussion by bsoisthp** [utmapp/UTM#4410] [UTM Help (macOS)]
Not able to proceed further as it needs Internet to proceed - not sure about the configuration, please suggest .
<img width="823" alt="No Options available- need Internet access" src="https://user-images.githubusercontent.com/42204425/189806001-6f333758-8daa-4190-a343-06935dc612a3.png">

**Comment by ghost** [comment]
if your host has internet, but not the VM, go to the VM settings, and use bridge mode, not shared, or better yet, Emulated VLAN.
"The solution is to use Emulated VLAN, that will use port forwarding from the host machine." so host VPN settings apply to VM.
	https://github.com/utmapp/UTM/issues/3245

If you got no internet on the host, and
If Windows 11 setup is stuck due to lack of network connection:

    Press Shift + F10 to launch Command Prompt.
    Type in OOBE\BYPASSNRO and press Enter.
    Your VM should reboot and at the setup screen you should see an option for “I don’t have internet.”
    Once Windows setup is completed, make sure to install the SPICE guest tools for network drivers.

from end of page : https://mac.getutm.app/gallery/windows-10
